proto/minder/v1/minder.proto

```diff
@@ -2026,8 +2026,7 @@ enum ProviderType {
     PROVIDER_TYPE_GIT = 3 [(name) = "git"];
     PROVIDER_TYPE_OCI = 4 [(name) = "oci"];
     PROVIDER_TYPE_REPO_LISTER = 5 [(name) = "repo-lister"];
-    PROVIDER_TYPE_DOCKERHUB = 6 [(name) = "dockerhub"];
-    PROVIDER_TYPE_GHCR = 7 [(name) = "ghcr"];
+    PROVIDER_TYPE_CONTAINER_LISTER = 6 [(name) = "container-lister"];
```

Comment:
Ah great I think this answers my earlier question about the type (reviewing patches one by one)